### PR TITLE
Clarify restriction of domain applies to team and user account creation

### PR DESCRIPTION
**Pull request by it33**

### CODE REVIEW

**Comment by jwilander** [comment]
+1

File: web/react/components/team_general_tab.jsx [hunk 1]
```diff
@@ -424,7 +424,7 @@ export default class GeneralTab extends React.Component {
                             </div>
                         </div>
                     </div>
-                    <div className='setting-list__hint'>{'Your Invite Code is used in the URL sent to people to join your team. Regenerating your Invite Code will invalidate the URLs in previous invitations, unless "Allow anyone to sign-up from login page" is enabled.'}</div>
+                    <div className='setting-list__hint'>{'Regenerating your Invite Code will invalidate the sign-up links created by the "Get Team Invite Link"  option in the main menu, as well as links sent in email via the "Invite New Member" menu option.'}</div>
```
Comment (esethna) [inline web/react/components/team_general_tab.jsx]: I just tested this and regenerating the invite code only invalidates the "Get team invite link" url, but the email that comes from "Invite New Member" still worked after regenerating the invite code, regardless of the setting for "Allow anyone to sign-up from login page"

Comment (it33) [inline web/react/components/team_general_tab.jsx]: Thanks @eric / @esethna, could I ask your help to: 

1. Take the changes from this PR and merge them into your branch for https://github.com/mattermost/platform/pull/1794? 
2. Correct the wording as you feel appropriate? 

You're probably in a better spot to get this fully correct. 

I can close my PR after you've got yours ready?

Comment (esethna) [inline web/react/components/team_general_tab.jsx]: Done, thanks @it33. You can close this PR.

**Pull request closed without merging**